tasks/pages/organization.tsx

```diff
@@ -0,0 +1,257 @@
+    onMutate: async(organization: OrganizationDTO) => {
+      await queryClient.cancelQueries({ queryKey: [queryKey] })
+      const previousOrganizations = queryClient.getQueryData<OrganizationDTO[]>([queryKey])
+      queryClient.setQueryData<OrganizationDTO[]>(
+        [queryKey],
+        (old) => [...(old === undefined ? [] : old.filter(value => value.id !== organization.id)), organization])
+      organizations.sort((a, b) => a.longName.localeCompare(b.longName))
+      return { previousOrganizations }
+    },
+    onError: (_, newTodo, context) => {
+      queryClient.setQueryData([queryKey], context === undefined ? [] : context.previousOrganizations)
+    },
+    onSettled: () => {
+      queryClient.invalidateQueries({ queryKey: [queryKey] }).then()
+    },
+  })
+
+  const deleteMutation = useMutation({
+    mutationFn: async (organization) => {
+      // TODO create request for organization
+      organizations = [...organizations.filter(value => value.id !== organization.id)]
+      setSelectedOrganization(undefined)
+    },
+    onMutate: async(organization: OrganizationDTO) => {
+      await queryClient.cancelQueries({ queryKey: [queryKey] })
+      const previousOrganizations = queryClient.getQueryData<OrganizationDTO[]>([queryKey])
+      queryClient.setQueryData<OrganizationDTO[]>(
+        [queryKey],
+        (old) => [...(old === undefined ? [] : old.filter(value => value.id !== organization.id))])
+      return { previousOrganizations }
+    },
+    onError: (_, newTodo, context) => {
+      queryClient.setQueryData([queryKey], context === undefined ? [] : context.previousOrganizations)
+    },
+    onSettled: () => {
+      queryClient.invalidateQueries({ queryKey: [queryKey] }).then()
+    },
+  })
+
+  const { isLoading, isError, data, error } = useQuery({
+    queryKey: [queryKey],
+    queryFn: async () => {
+      // TODO fetch user organizations
+      return organizations
+    },
+  })
+
+  if (!user) return null
+
+  // TODO add view for loading
+  if (isLoading) {
+    return <div>Loading Widget</div>
+  }
+
+  // TODO add view for error or error handling
+  if (isError) {
+    return <div>Error Message</div>
+  }
+
+  return (
+    <div className={tw('w-screen h-screen flex flex-col')}>
+      <Head>
+        <title>{translation.organizations}</title>
+      </Head>
+
+      <Header
+        title="helpwave"
+        navigation={[
+          { text: 'Dashboard', href: '/' },
+          { text: 'Contact', href: '/contact' },
+        ]}
+        actions={[<UserMenu key="user-menu" user={user}/>]}
+      />
+      <TwoColumn
+        left={(
+          <OrganizationDisplay
+            selectedOrganization={selectedOrganization}
+            organizations={data as OrganizationDTO[]}
+            onSelectionChange={setSelectedOrganization}
+          />
+        )}
+        right={(
+          <OrganizationDetail
+            key={selectedOrganization === undefined ? 'undefined' : selectedOrganization.longName}
```

Comment:
To prevent confusions with the `undefined` type. `key` is used for reacts re-rendering?
```suggestion
            key={selectedOrganization === undefined ? 'unselected' : selectedOrganization.longName}
```